pkg/constants/constants.go

```diff
@@ -136,6 +136,7 @@ const (
 // feature flog constants
 const (
 	VSphereLocalModeFlag        = "local-mode"
+	VSphereLocalModeFeature     = "EnableLocalMode"
```

Comment:
What is VSphereLocalModeFlag used for?

Reply:
The master branch PR is committed - #225 

VSphereLocalModeFlag is used internally in the code and also in the Config-map created. This flag should not be set by the user.